src/components/PupilComponents/BrowseTierSelector/BrowseTierSelector.tsx

```diff
@@ -9,36 +9,28 @@ import { resolveOakHref } from "@/common-lib/urls";
 export type TierData = Pick<
   ProgrammeFields,
   "tier" | "tierSlug" | "tierDisplayOrder" | "tierDescription"
->;
+> & {
+  isLegacy: boolean;
+};
 export const BrowseTierSelector = ({
   tiers,
   baseSlug,
   examboardSlug,
-  isLegacy,
   phaseSlug,
 }: {
   tiers: TierData[];
   baseSlug: string;
   examboardSlug?: string | null;
-  isLegacy: boolean;
   phaseSlug: PupilProgrammeListingData["programmeFields"]["phaseSlug"];
 }) => {
-  const orderedTiers = tiers.sort((a, b) => {
-    if (a.tier && b.tier) {
-      if (a.tier < b.tier) {
-        return -1;
-      }
-      if (a.tier > b.tier) {
-        return 1;
-      }
-    }
-    return 0;
-  });
+  const orderedTiers = tiers.sort(
+    (a, b) => (a.tierDisplayOrder ?? 0) - (b.tierDisplayOrder ?? 0),
+  );
 
   const programmeSlugs = orderedTiers.map(
     (tier) =>
       `${baseSlug}-${tier.tierSlug}${examboardSlug ? `-${examboardSlug}` : ""}${
-        isLegacy ? "-l" : ""
+        tier.isLegacy ? "-l" : ""
```

Comment:
we have a collection of slugModifiers in utils which includes a few functions for constructing / validating legacy URLs, might be worth looking to see if any are useful

Reply:
@njblore Couldn't see one that would help for this, I think isSlugLegacy could be used in one spot though. Thanks